.github/workflows/test.yml

```diff
@@ -13,13 +13,16 @@ jobs:
     runs-on: ubuntu-22.04
     timeout-minutes: 20
     steps:
-      - uses: actions/setup-go@v3
-        with:
-          go-version: 1.19.x
       - uses: actions/checkout@v3.3.0
         with:
           path: src/github.com/containerd/nerdctl
           fetch-depth: 100
+      - uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.x
+          check-latest: true
+          cache: true
+          cache-dependency-path: src/github.com/containerd/nerdctl
```

Comment:
would you please set `cache-dependency-path` for other jobs

Reply:
We do not need that as We do not need it as src/github.com/containerd/nerdctl is mapped at checkout actions in project checks only.